duneanaobj/StandardRecord/SRNDTrackMatch.h

```diff
@@ -0,0 +1,19 @@
+/// \author  J. Wolcott <jwolcott@fnal.gov> & F. Akbar <fakbar@ur.rochester.edu>
+/// \date    Nov. 2021
+
+#ifndef DUNEANAOBJ_SRNDTRACKMATCH_H
+#define DUNEANAOBJ_SRNDTRACKMATCH_H
+
+namespace caf
+{
+  class SRNDTrackMatch
+  {
+    public:
+      int larTrkIdx       = -1;
+      int tmsTrkIdx       = -1;
+      float xverseDispl   = -999.;
```

Comment:
`xverse` was nontransparent to me for a minute there

Reply:
do you find `transdispl` more ... transparent?

Reply:
Yes, I think so.

Reply:
ok (d0cf7448)